src/display.js

```diff
@@ -0,0 +1,120 @@
+import {
+  divFactory,
+  buttonFactory,
+  headerFactory,
+  paragraphFactory
+} from "./elementfactory";
+
+const displayWeatherData = (data, unit) => {
+  const cityName = document.getElementById("citytitle");
+  const content1 = document.getElementById("content1");
+  const content2 = document.getElementById("content2");
+  const card = document.getElementsByClassName("card")[0];
```

Comment:
When you want to find the first element by a classname you can use querySelector:
```
const card = document.querySelector('.card')
```
And it also works with any CSS selector.